build-system/tasks/karma.conf.js

```diff
@@ -139,55 +147,55 @@ module.exports = {
     // SauceLabs configurations.
     // New configurations can be created here:
     // https://wiki.saucelabs.com/display/DOCS/Platform+Configurator#/
-    SL_Chrome_latest: {
+    SL_Chrome_latest: Object.assign({
       base: 'SauceLabs',
       browserName: 'chrome',
       version: 'latest',
-    },
-    SL_Chrome_android: {
+    }, SAUCE_TIMEOUT_CONFIG),
+    SL_Chrome_android: Object.assign({
       base: 'SauceLabs',
       browserName: 'android',
       version: 'latest',
-    },
-    SL_Chrome_45: {
+    }, SAUCE_TIMEOUT_CONFIG),
+    SL_Chrome_45: Object.assign({
       base: 'SauceLabs',
       browserName: 'chrome',
       version: '45',
-    },
-    SL_Android_latest: {
+    }, SAUCE_TIMEOUT_CONFIG),
+    SL_Android_latest: Object.assign({
```

Comment:
Somewhat relatedly. I've noticed that most of our timeouts happen with Chrome on Android (the one that says Mobile Webview) and Chrome on Windows. I don't know if it's the case that those should be longer, or are we just more broken on said platforms.

Reply:
It's possible there's more than one thing at play. Let's merge this and observe the results.